doc/api/process.md

```diff
@@ -1982,6 +1982,19 @@ A boolean value that is `true` if the current Node.js build includes support for
 
 ***
 
+## `process.features.typescript`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+* {boolean|string}
+
+A string / boolean value that is `"strip"` if Node.js is run with `--experimental-strip-types`,
+`"transform"` if Node.js is run with `--experimental-transform-types`, and `false` otherwise.
+
+***
```

Comment:
we are not using divider

Reply:
The rest of the `process.features` items have the divider, should I remove it anyway?

Reply:
Thats weird I dont remember adding it 😅😅🤣🤣 nvm then

Reply:
The other ones were removed in https://github.com/nodejs/node/pull/55149.